test/cpp/api/optim_baseline.h

```diff
@@ -7,75 +7,351 @@
 namespace expected_parameters {
 
 static std::vector<std::vector<torch::Tensor>> Adam = {
+  {
```

Comment:
Might be valuable to comment how you generated these?

Reply:
There's `// @generated from test/cpp/api/optim_baseline.py` at the top :)

Reply:
Oops, thanks!